src/main.cpp

```diff
@@ -984,6 +996,11 @@ QString UsageArguments ( char** argv )
            "  -w, --welcomemessage  welcome message to display on connect\n"
            "                        (string or filename)\n"
            "  -z, --startminimized  start minimizied\n"
+           "      --serverpublicip  specify your public IP address when\n"
+           "                        running a slave and your own directory server\n"
+           "                        behind the same NAT\n"
+           "      --serverbindip    specify the IP address the server will bind to\n"
+           "      --notifyserver    specify server to receive UDP notifications IP:PORT\n"
```

Comment:
I'd suggest grabbing the short option `-N` and inserting this in the right order in the help text (alphabetical by short option, then any closely related options without short options).